delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/config/AggregateConfig.scala

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.config
+
+import ch.epfl.bluebrain.nexus.sourcing._
+import ch.epfl.bluebrain.nexus.sourcing.processor.{EventSourceProcessorConfig, StopStrategyConfig}
+
+/**
+  * Aggregate configuration.
+  *
+  * @param stopStrategy     the stop strategy configuration for this aggregate [[EventDefinition]]
+  * @param snapshotStrategy the snapshot strategy configuration for this aggregate [[EventDefinition]]
+  * @param processor        the event source processor config
```

Comment:
Remove this. Is exists now in `ch.epfl.bluebrain.nexus.sourcing.AggregateConfig`